gis/qlandkartegt/Portfile

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+oick 440cd9d0c5b qlandkartegt: upgrade to 1.8.1-macports-20200922
```

Comment:
Stray line causes portfile parsing to fail

Reply:
done